system/jlib/jsocket.cpp

```diff
@@ -6742,6 +6742,7 @@ int wait_multiple(bool isRead,               //IN   true if wait read, false it
 #ifdef _DEBUG
         DBGLOG("%s",dbgSB.str());
 #endif
+        res = readySocks.ordinality();
```

Comment:
I think line 6731 also needs changing to match this, to remove the check for POLLNVAL.

Reply:
it was ignoring POLLNVAL before and after this change.

Except before it would have ignore it (not filled in readySocks),
and return >0.
The caller would then hit an assert looking at which of the readySocks had changed.

So same issue as the incident I hit (not with POLLNVAL) but with revents=0.
Namely the returned number of ready socks msmatched the number filled in readySocks.

I think you're right that POLLNVAL should not be ignored,
but if we are going to start returning it (without causing assert to be hit), then it should a separate JIRA/PR so we can track it, in case it has any consequences.

Reply:
@mckellyln - can you link the 2 JIRA's together to more easily find them later? Thanks.